src/treq/api.py

```diff
@@ -81,6 +81,9 @@ def request(method, url, **kwargs):
     :param data: Optional request body.
     :type data: str, file-like, IBodyProducer, or None
 
+    :param json: Optional JSON-serializable content to pass in body.
+    :type json: dict, list/tuple, int, string/unicode/bytearray, bool, or None
```

Comment:
`bytearray` is not JSON-serializable (I tested 2.7 and 3.5), so it should be removed from this list.